using-mobile-extensions/adobe-campaign-standard-beta.md

```diff
@@ -29,7 +26,7 @@ Trying to find your ACS endpoint URLs? Contact your beta manager.
 Provide endpoint URL\(s\) for your Adobe Campaign Standard instances. You may specify up to three unique endpoints for your development, staging, and production environments. 
 
 {% hint style="warning" %}
-For this beta extension, these endpoints URLs should be entered in **without** a prefixing  `http://` or `https://.`
+For this extension, these endpoints URLs should be entered in **without** a prefixing  `http://` or `https://.`and must not end with a forward slash.
```

Comment:
Please change the sentence to read as follows:

"For this extension, these endpoint URLs should be entered **without** the `http://` or `https://.`and cannot end with a forward slash."